organizational/conduct.rst

```diff
@@ -0,0 +1,88 @@
+Code of Conduct
+===============
+
+Hacker Society [#hacsoc]_ is, as Tim wrote in :doc:`hacsoc`, about an open
+society:
+
+  An open society promotes freedom of thought and expression, personal
+  responsibility, individualism, cooperation, and equality of all citizens.
+
+As such, we expect that our members behave in a way that allows us to achieve
+this.  Participants in HacSoc events are expected to treat each other with
+dignity and respect for each other's perspectives, experiences, and knowledge.
+In particular, participants should recognize that people of all skill levels are
+welcome, and should always be willing to help others learn.
+
+We want Hacker Society to be a place where people can be comfortable regardless
+of gender, gender identity and expression, sexual orientation, disability,
+physical appearance, body size, race, age, or religion.  As such, we do not
```

Comment:
To support Hacker Society's mission of: learning technology from each other,
building community, and promoting equality, Hacker Society must be a place
where people can be comfortable regardless of gender, gender identity and
expression, sexual orientation, disability, physical appearance, body size,
race, age, or religion. Therefore, we do not tolerate: